src/pages/Login/Login.jsx

```diff
@@ -24,12 +34,32 @@ function Login() {
                 .required(t("Required"))
                 .matches(regexPassword, t("Password must be 7-19 charactors and contain at least one letter, one number and a special charactors"))
         }),
-        onSubmit: (formValue) => {
+        onSubmit: async (formValue) => {
+            try {
+                const {email, password} = formik.values
+                const userFirebase = await signInWithEmailAndPassword(
+                    auth, 
+                    email, 
+                    password
+                )
+                const user = userFirebase.user
+                await dispatch(getUserDetailDbJson(user.uid))
+                dispatch(setNotification({type: 'success', message: t('Login Success')}))
+                navigate('/')
+            } catch (err) {
+                dispatch(setNotification({type: 'error', message: t('Login Failed')}))
+                return err.message
+            }
         }
     })
 
     return (
         <section className={styles['form-login']}>
+            <div className={styles["toast"]}>
```

Comment:
```suggestion
            <div className={styles['toast']}>
```
Đồng bộ code của mình lên nhé, đừng có chỗ thì " chỗ thì '